### ISSUE TITLE
Wunderbar no longer visible after update [6.7.2 -> 6.7.3]

### ISSUE THREAD

**Issue by Juanbreezer** [Toxicom/toxiui#110]
### Description
After updating from 6.7.2 to 6.7.3 on Cataclysm Classic the WunderBar module is not loading at all anymore and generates the below mentioned LUA error. The same error occurs with a /reload and full game restart.

### Lua error
```lua
1x ...lvUI_ToxiUI/Modules/WunderBar/SubModules/DataBar.lua:70: attempt to call upvalue 'C_Reputation_GetWatchedFactionData' (a nil value)
[string "@ElvUI_ToxiUI/Modules/WunderBar/SubModules/DataBar.lua"]:70: in function `callback'
[string "@ElvUI_ToxiUI/Core/Functions/Event.lua"]:285: in function `ContinueOutOfCombat'
[string "@ElvUI_ToxiUI/Modules/WunderBar/SubModules/DataBar.lua"]:68: in function `OnEvent'
[string "@ElvUI_ToxiUI/Modules/WunderBar/SubModules/DataBar.lua"]:289: in function `OnWunderBarUpdate'
[string "@ElvUI_ToxiUI/Modules/WunderBar/SubModules/DataBar.lua"]:520: in function `OnInit'
[string "@ElvUI_ToxiUI/Modules/WunderBar/Modules.lua"]:278: in function `UpdatePanelSubModules'
[string "@ElvUI_ToxiUI/Modules/WunderBar/Modules.lua"]:368: in function `ConstructModules'
[string "@ElvUI_ToxiUI/Modules/WunderBar/Core.lua"]:163: in function <...rfaceElvUI_ToxiUI/Modules/WunderBar/Core.lua:161>
[string "@ElvUI_ToxiUI/Core/Functions/Event.lua"]:252: in function <ElvUI_ToxiUI/Core/Functions/Event.lua:250>
[string "=[C]"]: ?
[string "@ElvUI_ToxiUI/Core/Functions/Event.lua"]:170: in function <ElvUI_ToxiUI/Core/Functions/Event.lua:169>
[string "=[C]"]: ?
...
[string "@ElvUI_ToxiUI/Core/Core.lua"]:128: in function `callback'
[string "@ElvUI_ToxiUI/Core/Functions/Event.lua"]:312: in function `ContinueAfter'
[string "@ElvUI_ToxiUI/Core/Functions/Event.lua"]:351: in function `ContinueAfterElvUIUpdate'
[string "@ElvUI_ToxiUI/Core/Core.lua"]:104: in function <ElvUI_ToxiUI/Core/Core.lua:102>
[string "@ElvUI_ToxiUI/Core/Functions/Event.lua"]:379: in function <ElvUI_ToxiUI/Core/Functions/Event.lua:374>
[string "=[C]"]: ?
[string "@ElvUI_ToxiUI/Core/Functions/Event.lua"]:170: in function <ElvUI_ToxiUI/Core/Functions/Event.lua:169>
[string "=[C]"]: ?
[string "@ElvUI_ToxiUI/Core/Functions/Event.lua"]:173: in function `TriggerEvent'
[string "@ElvUI_ToxiUI/Core/Functions/Event.lua"]:388: in function <ElvUI_ToxiUI/Core/Functions/Event.lua:387>

Locals:
(*temporary) = nil
(*temporary) = "attempt to call upvalue 'C_Reputation_GetWatchedFactionData' (a nil value)"
self = <table> {
 OnInit = <function> defined @ElvUI_ToxiUI/Modules/WunderBar/SubModules/DataBar.lua:495
 OnEnter = <function> defined @ElvUI_ToxiUI/Modules/WunderBar/SubModules/DataBar.lua:258
 barIcon = FontString {
 }
 Enable = <function> defined @ElvUI_Libraries/Core/Ace3/AceAddon-3.0/AceAddon-3.0.lua:290
 NewModule = <function> defined @ElvUI_Libraries/Core/Ace3/AceAddon-3.0/AceAddon-3.0.lua:231
 Data = <table> {
 }
 EnableModule = <function> defined @ElvUI_Libraries/Core/Ace3/AceAddon-3.0/AceAddon-3.0.lua:328
 modules = <table> {
 }
 GetModule = <function> defined @ElvUI_Libraries/Core/Ace3/AceAddon-3.0/AceAddon-3.0.lua:206
 IterateEmbeds = <function> defined @ElvUI_Libraries/Core/Ace3/AceAddon-3.0/AceAddon-3.0.lua:438
 defaultModuleLibraries = <table> {
 }
 OnClick = <function> defined @ElvUI_ToxiUI/Modules/WunderBar/SubModules/DataBar.lua:277
 const = <table> {
 }
 mode = 0
 UpdateTooltip = <function> defined @ElvUI_ToxiUI/Modules/WunderBar/SubModules/DataBar.lua:244
 updateRepNextOutOfCombat = false
 UpdateSmartMode = <function> defined @ElvUI_ToxiUI/Modules/WunderBar/SubModules/DataBar.lua:387
 bar = StatusBar {
 }
 SetDefaultModulePrototype = <function> defined @ElvUI_Libraries/Core/Ace3/AceAddon-3.0/AceAddon-3.0.lua:405
 barFrame = Button {
 }
 Module = TXUIWunderBarLeftPanelModule3 {
 }
 UpdatePosition = <function> defined @ElvUI_ToxiUI/Modules/WunderBar/SubModules/DataBar.lua:324
 name = "ElvUI_ElvUI_ToxiUI_WunderBar_DataBar"
 noData = false
 IsEnabled = <function> defined @ElvUI_Libraries/Core/Ace3/AceAddon-3.0/AceAddon-3.0.lua:447
 orderedModules = <table> {
 }
 GetValues = <function> defined @ElvUI_ToxiUI/Modules/WunderBar/SubModules/DataBar.lua:43
 DisableModule = <function> defined @ElvUI_Libraries/Core/Ace3/AceAddon-3.0/AceAddon-3.0.lua:346
 ThrowError = <function> defined @ElvUI_ToxiUI/Core/Functions/Log.lua:85
 SetDefaultModuleLibraries = <function> defined @ElvUI_Libraries/Core/Ace3/AceAddon-3.0/AceAddon-3.0.lua:363
 frame = Frame {
 }
 OnLeave = <function> defined @ElvUI_ToxiUI/Modules/WunderBar/SubModules/DataBar.lua:268
 SubModuleHolder = Frame {
 }
 LogInfo = <function> defined @ElvUI_ToxiUI/Core/Functions/Log.lua:147
 data = <table> {
 }
 baseName = "ElvUI_ToxiUI"
 LogTrace = <function> defined @ElvUI_ToxiUI/Core/Functions/Log.lua:157
 LogDebug = <function> defined @ElvUI_ToxiUI/Core/Functions/Log.lua:152
 LogWarning = <function> defined @ElvUI_ToxiUI/Core/Functions/Log.lua:142
 IsModule = <function> defined @ElvUI_Libraries/Core/Ace3/AceAddon-3.0/AceAddon-3.0.lua:213
 db = <table> {
 }
 SetDefaultModuleState = <function> defined @ElvUI_Libraries/Core/Ace3/AceAddon-3.0/AceAddon-3.0.lua:383
 UpdateElements = <function> defined @ElvUI_ToxiUI/Modules/WunderBar/SubModules/DataBar.lua:402
 UpdateExperienceTooltip = <function> defined @ElvUI_ToxiUI/Modules/WunderBar/SubModules/DataBar.lua:190
 SetEnabledState = <function> defined @ElvUI_Libraries/Core/Ace3/AceAddon-3.0/AceAddon-3.0.lua:420
 OnWunderBarUpdate = <function> defined @ElvUI_ToxiUI/Modules/WunderBar/SubModules/DataBar.lua:284
 enabledState = true
 UpdateInfoText = <function> defined @ElvUI_ToxiUI/Modules/WunderBar/SubModules/DataBar.lua:421
 CreateBar = <function> defined @ElvUI_ToxiUI/Modules/WunderBar/SubModules/DataBar.lua:429
 infoText = FontString {
 }
 updateExpNextOutOfCombat = false
 OnEvent = <function> defined @ElvUI_ToxiUI/Modules/WunderBar/SubModules/DataBar.lua:56
 UpdateBar = <function> defined @In
```

### ToxiUI Status Report
![image](https://github.com/user-attachments/assets/76b0a35f-a3d4-42d4-82ec-14a8403716e3)

### Screenshots
![image](https://github.com/user-attachments/assets/d6437daa-a548-4efe-b860-61d0f8a761ef)

### To reproduce
1. Update via the CurseForge client.
2. Start game and/or /reload

**Comment by Toxicom** [comment]
Thanks, I know why this happens. Will be fixed next patch.

**Comment by Toxicom** [comment]
Should be fixed in [v6.7.4](https://github.com/Toxicom/toxiui/releases/tag/v6.7.4)
